Fabric.Authorization.API/Services/IdentitySearchService.cs

```diff
@@ -74,9 +73,15 @@ public class IdentitySearchService
             var groupEntities = new List<Group>();
             foreach (var groupId in groupIds)
             {
-                _logger.Debug($"getting groupId {groupId}");
-                var group = await _groupService.GetGroup(groupId, request.ClientId);
-                groupEntities.Add(group);
+                try
+                {
+                    var group = await _groupService.GetGroup(groupId, request.ClientId);
+                    groupEntities.Add(group);
+                }
+                catch (NotFoundException<Group> ex)
+                {
+                    _logger.Error($"{ex.Message} (Group is mapped to at least 1 valid role)");
```

Comment:
is this error message correct? 

>  (Group is mapped to at least 1 valid role)

Reply:
@kpaul, yes, the query above gets all valid roles owned by the client and pulls the Groups out of those roles.